collector/logs/sources/journal/tailer_linux.go

```diff
@@ -25,6 +25,17 @@ const (
 	journald_line_break_value_line_max = "line-max"
 )
 
+type openmode string
+
+// constants for the mode we're opening the journal in
+const (
+	// journal_open_mode_startup is the initial path we use for opening. Attempts to seek to the last read entry, or to the head if no cursor is found.
+	journal_open_mode_startup openmode = "startup"
```

Comment:
Go style suggests mixedCase instead of snake case.  https://go.dev/doc/effective_go#mixed-caps